src/singletons/Settings.hpp

```diff
@@ -247,6 +247,8 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/streamerMode/supressLiveNotifications", false};
     BoolSetting streamerModeSuppressInlineWhispers = {
         "/streamerMode/suppressInlineWhispers", true};
+    BoolSetting streamerModeSuppressModActions = {
+        "/streamerMode/suppressModActions", true};
```

Comment:
where is consistency with the changelog & changes made to `GeneralPage.cpp`? I suggest to either keep suppress or hide everywhere. ~~really surprised nobody even cared/noticed~~
```suggestion
    BoolSetting streamerModeHideModActions = {
        "/streamerMode/hideModActions", true};
```

Reply:
Fair point, changed in f1c0d79.